### ISSUE TITLE
feature request: please implement -q quiet

### ISSUE THREAD

**Issue by hboetes** [ggreer/the_silver_searcher#1456]
Since the -q option is not implemented, my request is to add support for the -q or quiet output, which is very practical in programs.

**Comment by askreet** [comment]
Would this options just suppress all output? Lacking such an option, you can use:

    ag --ruby search_term >/dev/null

**Comment by hboetes** [comment]
Of course, I can use a workaround with > /dev/null 2>&1, but that's not the point of my request. The -q option is great for scripting.

**Comment by nebbish** [comment]
I was just looking for a '-q' option so I can use "ag" with "find":       (the exit code would matter, not the output)
```
find <path> -name <glob> -exec ag <expr> {} ; -exec <thing-I-want-to-launch-against-all-matching-files> {} ;
```

Maybe there is a better way?    I realize I'm launching a super fast utility repeatedly, and I'm sure it's loosing some of the efficiency built-into `ag`

On windows I might:   `for /f %r in ('ag -l <expr> <path>') do @( thing-i-want-to-launch %r )`
And on my Linux:   `for r in $(ag -l <expr> <path>); do echo <thing-i-want-to-launch> $r; done`

I like the cross-platform methodology using find.
